src/resources/PowerShellWorkerStrings.resx

```diff
@@ -352,7 +352,43 @@
   <data name="DependencySnapshotDoesNotContainAcceptableModuleVersions" xml:space="preserve">
     <value>Dependency snapshot '{0}' does not contain acceptable module versions.</value>
   </data>
+  <data name="ExternalDurableSDKName" xml:space="preserve">
+    <value>AzureFunctions.PowerShell.Durable.SDK</value>
+  </data>
+  <data name="InternalDurableSDKName" xml:space="preserve">
+    <value>Microsoft.Azure.Functions.PowerShellWorker</value>
+  </data>
+  <data name="TracePipelineObjectCommand" xml:space="preserve">
```

Comment:
This `TracePipelineObjectCommand` should not be here. You can add them as constant/variable in the class file itself, .e.g, https://github.com/Azure/azure-functions-powershell-worker/blob/ac95680f3d905a85352410d942f66e536d9ddde5/src/DependencyManagement/PowerShellGalleryModuleProvider.cs#L82 or https://github.com/Azure/azure-functions-powershell-worker/blob/ac95680f3d905a85352410d942f66e536d9ddde5/src/Utility/Utils.cs#L20

Reply:
In this case, I added it inline as we only use it in one place. I think that corresponds to your second example (just like "Import-Module" is hardcoded) but perhaps I misunderstood. Please let me know if this seems right to you:

https://github.com/Azure/azure-functions-powershell-worker/blob/7c24933ef941879b617fd69be90814cc5c3e45fe/src/DurableSDK/PowerShellServices.cs#L191-L194

Reply:
@Francisco-Gamino: just for my understanding, how should I determine what strings are appropiate for the PowerShellWorkerStrings.resx file?

Reply:
> @Francisco-Gamino: just for my understanding, how should I determine what strings are appropiate for the PowerShellWorkerStrings.resx file?

Great question. The `PowerShellWorkerStrings.resx` file contain strings that will be shown to the user.

Reply:
> Great question. The `PowerShellWorkerStrings.resx` file should only contain strings that will be displayed to the user.

@davidmrdavid -- I've just realized that I was missing some information in this statement: `Great question. The PowerShellWorkerStrings.resx file should only contain strings that will be displayed to the user.` In addition to this, the strings in the resx file will be translated to all supported languages (this is because PowerShell and the help files used to be shipped with the OS), so all strings in this files will be translated. Because of this, proper names like `AzureFunctions.PowerShell.Durable.SDK` and  `Microsoft.Azure.Functions.PowerShellWorker`  should not be included these files. 
Please make these two (`Microsoft.Azure.Functions.PowerShellWorker` and `AzureFunctions.PowerShell.Durable.SDK`) constants.